reporter/src/funTest/assets/bom.cyclonedx-NPM-%40ort-project-without-findings-1.0.json

```diff
@@ -0,0 +1 @@
+{"bomFormat":"CycloneDX","specVersion":"1.3","serialNumber":"urn:uuid:01234567-0123-0123-0123-01234567","version":1,"externalReferences":[{"type":"website","url":"https://github.com/oss-review-toolkit/ort"}]}
```

Comment:
Nit: Missing final newline.

Reply:
There is no issue in the sense of failing tests. It's just a "formal thing" to get of the nasty red marker that GitHub shows, as Git discourages to commit files without a final newline:

![image](https://user-images.githubusercontent.com/349154/173327332-1d7c683a-841d-4b7e-a75a-e3d187492630.png)

Reply:
Ok, I get it.